python/src/trezorlib/btc.py

```diff
@@ -74,10 +75,57 @@ def get_public_node(
             show_display=show_display,
             coin_name=coin_name,
             script_type=script_type,
+            ignore_xpub_magic=ignore_xpub_magic,
         )
     )
 
 
+def get_descriptor(
```

Comment:
this function should move to `trezorlib.cli.btc` as it's too high-level for this file

it should also probably accept `address_n`, and the `get-descriptor` command should also accept `-n/--address`, plus possibly `-a/--account` as a convenience shortcut

Reply:
I prefer this won't accept  `address_n` - I don't see a good reason why it should. Will move this to `trezorlib.cli.btc`

Reply:
If I moved this to `trezorlib.cli.btc`, where should I move unit test from 9af06e7de6b48f85efd32d9cd8a49d7276bc28da?

I really want to have this feature properly covered by tests.

Reply:
well the reason to accept `address_n` is to allow use by coins other than Bitcoin, but i don't care that much so eh

re tests, we can keep them where they are, importing from `trezorlib.cli` namespace is not discouraged or anything

i'm thinking that the bulk of the functionality should be tested in `python/tests` presumably by mocking out the `btc.get_public_node` call?
(i don't think it is super important though)

Reply:
Function moved to cli.btc in 15c3753

Tests moved in ec4d3c36aca2898a2ab1571f0766334c0fa13b9c